packages/core/drizzle/0100_lonely_revanche.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE "latitude"."metadata_type" ADD VALUE 'default';
```

Comment:
`default`? What about `hitl` or `human_in_the_loop` to be more clear?

Reply:
yeah we've talked about it and hitl is not good enough but will probably change it to "manual"